docker-entrypoint

```diff
@@ -13,9 +13,9 @@ APPRISE_URL=${APPRISE_URL:-""}
 
 # only use unix domain socket if no TCP endpoint is defined
 case "${DOCKER_SOCK}" in
-  "tcp://"*) HTTP_ENDPOINT="$(echo ${DOCKER_SOCK} | sed 's#tcp://#http://#')"
+  "tcp://"*) HTTP_ENDPOINT="$(echo "${DOCKER_SOCK}" | sed 's#tcp://#http://#')"
```

Comment:
might as well remove the useless echo here.

Reply:
Do you mean something like this?
```sh
"tcp://"*) HTTP_ENDPOINT="${DOCKER_SOCK//tcp:/http:}"
```

Then I get this warning:
> In POSIX sh, string replacement is undefined.shellcheck[SC3060](https://www.shellcheck.net/wiki/SC3060)

Reply:
Oh right, this is POSIX sh. My bad. Ignore then :)